sematic/api/endpoints/external_resources.py

```diff
@@ -151,3 +158,127 @@ def save_resource_endpoint(user: Optional[User]) -> flask.Response:
     payload = dict(external_resource=record.to_json_encodable())
 
     return flask.jsonify(payload)
+
+
+@sematic_api.route("/api/v1/external_resources/all/clean_orphaned", methods=["POST"])
```

Comment:
I presume `/all/` ensure this does not conflict with the `/:id` endpoint.
I wonder if there aren't ways to still do `/external_resources/clean_orphaned`. Like maybe if this endpoint is declared after the `/:id` one...
Ideally, can we do `/api/v1/external_resources/clean`?

Alternatively, we can have a
```GET /api/v1/external_resources?orphaned=1```
endpoint, then we call
```DELETE /api/v1/external_resources/:id```
on each of them.

That does generates more back-and-forth, but maybe more restful and modular?

Also, this would make the deactivation logic simpler since you'd only be deactivating one resource at a time.
For example, you wouldn't need `StateChanges`, you would just return a string (one of the possible state changes).